Web/starter_project_vanguard/styles/globals.css

```diff
@@ -21,7 +21,7 @@ body {
   background: linear-gradient(
       to bottom,
       transparent,
-      rgb(var(--background-end-rgb))
+      rgb(var(--background-end-rgb))    
```

Comment:
please remove the extra space here 
let's not track changes that don't involve your feature